peachjam_ml/urls.py

```diff
@@ -1,14 +1,19 @@
-from django.urls import re_path
+from django.urls import path, re_path
 from django.views.decorators.cache import cache_page
 
 from peachjam.urls import CACHE_DURATION
 
-from .views import SimilarDocumentsView
+from .views import SimilarDocumentsDocumentDetailView, SimilarDocumentsFolderView
 
 urlpatterns = [
     re_path(
         r"^(?P<frbr_uri>akn/.*)/similar-documents$",
-        cache_page(CACHE_DURATION)(SimilarDocumentsView.as_view()),
+        cache_page(CACHE_DURATION)(SimilarDocumentsDocumentDetailView.as_view()),
         name="document_similar_docs",
     ),
+    path(
+        "folder/<int:pk>/similar-documents",
+        cache_page(CACHE_DURATION)(SimilarDocumentsFolderView.as_view()),
```

Comment:
Do we want to cache this? If I add a document or two to my folder, then the suggestions must update.

Reply:
It's also not horribly expensive.

Reply:
makes sense, removed